src/components/SignUp/SignUp.scss

```diff
@@ -1,3 +1,13 @@
 .sign-up {
     margin: 2rem;
+    .error-red {
```

Comment:
same comment, try to make 1 general style between this & the form messages to avoid code duplication